File: Java/src/Ex3.java
public class Ex3 {

    static void printSubstring(String s, char ch) {
        int index = s.indexOf(ch);
        while (index > -1) {
            System.out.println(s.substring(index));
            index = s.indexOf(ch, index + 1);
        }
    }

    public static void main(String[] args) {
        printSubstring("miaom", 'm');

    }
}

File: python/ex3.py
def print_substring(s: str, ch: str) -> None:
    index = s.find(ch)
    while index > -1:
        print(s[index:])
        index = s.find(ch, index+1)


def main():
    print_substring("miaom", "m")


if __name__ == '__main__':
    main()
